### DISCUSSION TITLE
Set deletedBy in hook when using destroying records with paranoid

### DISCUSSION THREAD

**Discussion by toymachiner62** [sequelize/sequelize#14594] [Help & Questions]
## Issue Creation Checklist

[x] I have read the [contribution guidelines](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md)

## Issue Description

I want to have a column called `deletedBy` that is used to set a user id of who deleted the record. I'm using the paranoid option so they are soft deletes.

I found [this](https://github.com/sequelize/sequelize/issues/10497) issue which tries to do something similar. 

Is there a way to use `.destroy()` and set the user id for `deletedBy`?

### Is this issue dialect-specific?

- [x] No. This issue is relevant to Sequelize as a whole.

### Would you be willing to resolve this issue by submitting a Pull Request?

- [x] Yes, I have the time but I don't know how to start, I would need guidance.

**Comment by luizphelip3** [comment]
Hi! I have this same problem now, but I didn't find any answer in all internet. Do you have any idea about how to do it using only one query?
I'm trying to use .update() setting deletedAt column value = new Date() but this don't work.
